forge/routes/auth/permissions.js

```diff
@@ -13,17 +13,20 @@ const defaultPermissions = {
     'team:user:remove': { description: 'Remove Member', role: Roles.Owner, self: true },
     'team:user:change-role': { description: 'Modify Member role', role: Roles.Owner },
     // Projects
+    'project:member': { description: 'Create Project', role: Roles.Member },
```

Comment:
This permission is not needed - see comment on `routes/api/project.js` for explanation
```suggestion
```

Reply:
ahha - of course. Well spotted :)